sql/core/src/test/scala/org/apache/spark/sql/execution/local/HashJoinNodeSuite.scala

```diff
@@ -19,10 +19,12 @@ package org.apache.spark.sql.execution.local
 
 import org.apache.spark.sql.SQLConf
 import org.apache.spark.sql.catalyst.dsl.expressions._
-import org.apache.spark.sql.execution.joins.{BuildLeft, BuildRight, BuildSide}
+import org.apache.spark.sql.catalyst.expressions.{UnsafeProjection, Expression}
+import org.apache.spark.sql.execution.joins.{HashedRelation, BuildLeft, BuildRight, BuildSide}
+import org.apache.spark.sql.test.SharedSQLContext
 
 
-class HashJoinNodeSuite extends LocalNodeTest {
+class HashJoinNodeSuite extends LocalNodeTest with SharedSQLContext {
```

Comment:
where does this test use `SQLContext`? We made an effort to not introduce this dependency in an earlier patch so we should try to avoid the same here. #8764